### DISCUSSION TITLE
How can I get involved?

### DISCUSSION THREAD

**Discussion by chmac** [OpenHospitalityNetwork/openHospitality.network#50] [General]
Haven't gotten onto matrix yet, will do that. I see weekly meetings but only twice back in July. Still happening?

Background, I'm one of the founders of trustroots, and federation was always part of my vision for the platform, something that I'm now even more focused on. We always envisioned Trustroots as just a launchpad to get all kinds of other cool stuff out into the wild, but unfortunately our dreams were a bit unrealised, growth was slow, and so after a couple of years I stepped down from the board. But I'm super committed to federated and decentralised social tech as a core concept, and focusing most of my development time on that currently.

So, all that said, I'd love to find a way to connect with you folks and see if I can contribute. It's super inspiring to see you folks working on this. 🧡

**Comment by weex** [comment]
Hey @chmac welcome and thanks for creating the issue. Our meetings started out more formally but recently they're more hangouts as we're trying to keep it as asynchronous or I guess non-blocking as possible. It's amazing what Trustroots has done and become and I think most of us share the vision of decentralization at the core.

As far as contributing, you may see reference to [C4](https://rfc.zeromq.org/spec/42/) or collective code construction contract, a development flow is biased toward action. Merging is near automatic, as long as code claims to address a valid problem. That's why you see many "Problem:" issues. You're free to work on anything in the tracker or to add your own if you want to bring up any and all other roadblocks on the way toward the vision. I look forward to all discussion and presses of the merge button. If I were better in js I'd have done more myself but so far have kept it to sysadmin-type things. If you drop by matrix, please say hi (I'm weex2 there).

**Comment by mariha** [comment]
Hey @chmac! Great to see you here. 

One particular way you could contribute is by sharing with us your thoughts on federation or even more general helping us shape the vision of the project.

We have #33 which kind of describes what we plan to do, but it would be great to hear what were your ideas and how federation or decentralized hospex network could look like from your point of view?

I guess I happened to be an instigator of this project, was previously involved in WarmShowers Android app development, and with help from many others along the way: Saemy, Chagai an Nick, David, we got to where we are right now. I have a two years old kiddo what hugely limits my time capacity. I don't know JavaScript either but building on Java and jvm ecosystem familiarity I hope to find my way to be able to write some code too.

I am also a terrible organizer and luck leadership skills so wouldn't mind giving up some of project management / administrative / facilitation stuff, any feedback and help with how we could do better with the project is welcome. Even better - taking any initiative forward.

You are probably well familiar with Trustroots codebase so I can see you helping us find our way there, commenting on issues like this https://github.com/OpenHospitalityNetwork/fedi-trustroots/issues/16 with some tips what to consider and where to start. It's first one on bicycle touring community roadmap.

Code contributions are welcome too (of course!). We have two repositories, one where OHN website, Federated roadmap and general project things live and the other one where we built the actual federated platform by tweaking Trustroots codebase. 

To talk more in person, you can join our Mondays weekly meeting (after vacation it's back), our channel @ ohn:matrix.org or write to me directly @mariha:matrix.org

**Reply by chmac** [reply]
@mariha Joining a call sounds great. Are the Monday calls back online now? Otherwise I'd also be happy to setup a specific call. Is your gmail (on your GitHub profile) also a good way to reach out? I'm not yet on a matrix server.

**Reply by mariha** [reply]
Hey @chmac! Yes, Monday’s calls have been happening quite regularly for past few weeks. [In the pad](https://pad.kanthaus.online/ohn) you can find the coordinates and some notes from the last meetings. 

If that call didn’t work out for whatever reason then we can also have a[n ongoing] chat separately. 
Matrix is very convenient, I am at mariha:matrix.org. If you prefer email, I am using m.kozinska@gmail.com.

I am looking forward to talk to you!
Best,
Mariha

**Reply by chmac** [reply]
Cool, thanks. Do you know if there's a meeting next Monday? Today I had another commitment.

**Reply by mariha** [reply]
Yes, there is. If you prefer to meet sooner, tomorrow (Tuesday) I have nothing scheduled, Thursday and Friday morning at any time 10-2pm CET would be good too.

**Reply by chmac** [reply]
Awesome, next Monday works for me, happy to join then. Looking forward to it. Thanks for all the info.
